website/content/sdk/guides/intros/stocks/options.md

```diff
@@ -1,82 +1,110 @@
+| 425 | PG       | 2022-12-09 |    149   | Put    |      2.1 |   241 |  113 |  0.64 |  0.69 |
 
-Getting all option chains for the underlying symbol as a single Pandas DataFrame requires a little bit of scripting prior to analysis. This will query Tradier as the data source:
+### PCR
 
-```python
-from openbb_terminal.sdk import openbb
-import pandas as pd
-import numpy as np
+Get up to ten years of historical Put-Call Ratios.
 
-expirations = openbb.stocks.options.option_expirations(symbol = 'SPY')
-options_df: pd.DataFrame = []
+```python
+opt.pcr(start_date = '2012-01-01', window = 10, symbol = 'SPY')
+```
 
-for expirations in expirations:
-    options_df.append(openbb.stocks.options.chains(symbol = 'SPY', expiry = expirations))
+| Date                |    PCR |
+|:--------------------|-------:|
+| 2012-12-05 00:00:00 | 1.1815 |
+| 2012-12-06 00:00:00 | 1.7403 |
+| 2012-12-07 00:00:00 | 1.7023 |
+| 2012-12-10 00:00:00 | 1.8997 |
+| 2012-12-11 00:00:00 | 1.5384 |
+| 2022-11-28 00:00:00 | 1.2491 |
+| 2022-11-29 00:00:00 | 2.135  |
+| 2022-11-30 00:00:00 | 1.5901 |
+| 2022-12-01 00:00:00 | 0.9842 |
+| 2022-12-02 00:00:00 | 2.1346 |
 
-options_df = pd.concat(options_df)
+### Hist_CE
 
-options_df.set_index(keys = 'symbol', inplace = True)
+The historical daily data of an individual option, from ChartExchange, includes open interest.
 
-options_df
+```python
+opt.hist_ce(symbol = 'SPY', price = '400', date = '2023-01-20', call = False)
 ```
 
-Depending on the depth of options available, compiling the data may take upwards of thirty seconds.
-
-![DataFrame With all Expirations](https://user-images.githubusercontent.com/85772166/199893166-35ef062d-c16e-464d-a392-378ef4c6a1ee.png "DataFrame With all Expirations")
+|    | Date       |   Open |   High |   Low |   Close |   Change |   Volume |   Open Interest |   Change Since |
+|---:|:-----------|-------:|-------:|------:|--------:|---------:|---------:|----------------:|---------------:|
+|  0 | 2022-12-01 |   8.5  |  10.01 |  7.99 |    8.68 | -0.01251 |    11443 |           47690 |        0       |
+|  1 | 2022-11-30 |  14.44 |  15.68 |  8.79 |    8.79 | -0.3866  |    19763 |           46218 |       -0.01251 |
+|  2 | 2022-11-29 |  14.74 |  15.37 | 14.25 |   14.33 |  0.01415 |      666 |           45105 |       -0.39428 |
+|  3 | 2022-11-28 |  12.49 |  14.86 | 11.94 |   14.13 |  0.27297 |    10643 |           45828 |       -0.3857  |
+| 366 | 2021-05-21 |  37.72 |  37.72 | 37.59 |   37.59 | -0.01079 |        5 |               0 |       -0.76909 |
+| 367 | 2021-05-20 |  38    |  38    | 38    |   38    | -0.07317 |        2 |               0 |       -0.77158 |
+| 368 | 2021-05-19 |  41.67 |  41.67 | 41    |   41    |  0.07471 |       26 |               0 |       -0.78829 |
+| 369 | 2021-05-17 |  37.43 |  38.15 | 37.43 |   38.15 | -0.0453  |        3 |               0 |       -0.77248 |
+
+Stay tuned, more to come soon!
```

Comment:
Please remove this, we don't have to promise things.

Reply:
done